src/huggingface_hub/__init__.py

```diff
@@ -141,12 +142,13 @@ def __dir__():
             "from_pretrained_fastai",
             "push_to_hub_fastai",
         ],
-        "file_download": ["cached_download", "hf_hub_download", "hf_hub_url"],
```

Comment:
this is a backward incompatible change. We should either not move it, or if we're moving it, there should be a proxy function in this file to warn the users that it's been moved.